examples/next-js-netlify/Earthfile

```diff
@@ -0,0 +1,36 @@
+VERSION 0.6
+FROM --platform=linux/amd64 node:latest
+WORKDIR /app
+
+deps:
+    ARG auth_token
+    ARG site_id
+    # Set environment variables required for building and deploying to Netlify
+    ENV NETLIFY_AUTH_TOKEN=$auth_token
+    ENV NETLIFY_SITE_ID=$site_id
+    # Copy files required for building and deploying to Netlify
+    COPY netlify.toml ./
+    RUN npm install netlify-cli -g
```

Comment:
I would recommend doing the install first so that a change in the toml doesn't require re-downloading netlify-cli.

Reply:
I committed this change.